test/unit/mem/view/src/ViewSubViewTest.cpp

```diff
@@ -170,10 +212,15 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(
     using Elem = float;
     using Dim = alpaka::dim::Dim<TAcc>;
     using Size = alpaka::size::Size<TAcc>;
+    using Host = alpaka::acc::AccCpuSerial<Dim, Size>;
+    using DevHost = alpaka::dev::Dev<Host>;    
 
     using View = alpaka::mem::view::ViewSubView<Dev, Elem, Dim, Size>;
+    using Stream = alpaka::test::stream::DefaultStream<Dev>;    
 
+    DevHost devHost (alpaka::dev::DevMan<Host>::getDevByIdx(0));    
     Dev dev(alpaka::dev::DevMan<TAcc>::getDevByIdx(0u));
+    Stream stream (dev);
```

Comment:
I do not see a usage of those new variables and typedefs in this test case so this will only result in warnings for unused local variables / typedefs.